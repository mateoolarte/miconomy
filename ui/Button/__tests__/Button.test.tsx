import React from 'react';
import { render } from '@testing-library/react';
import { Button } from '..';

describe('Button', () => {
  it('renders correctly', () => {
    const { container } = render(<Button type="button">Button</Button>);

    expect(container.firstChild).toMatchSnapshot();
  });
});
