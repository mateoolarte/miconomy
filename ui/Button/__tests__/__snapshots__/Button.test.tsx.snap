// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<button
  class="chakra-button css-5c2176"
  type="button"
>
  Button
</button>
`;
